components/camel-graphql/src/main/java/org/apache/camel/component/graphql/GraphqlEndpoint.java

```diff
@@ -110,8 +112,12 @@ private CloseableHttpClient createHttpClient() {
             httpClientBuilder.setProxy(new HttpHost(hostname, port));
         }
         if (accessToken != null) {
+            String authType = "Bearer";
+            if(this.jwtAuthorizationType != null) {
```

Comment:
If you run a full build, this should be formatted to respect the camel code style standards.

Reply:
That missed my eye but should be fixed now